web/src/pages/roadmap.tsx

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+import ExternalLink from "@site/src/components/ExternalLink";
+import React from "react";
 
-export default function Home(): JSX.Element {
-  const url = 'https://github.com/orgs/tenzir/projects/8/views/1';
-  React.useEffect(() => {
-    window.location.replace(url)
-  }, [])
-  return <></>;
-}
+const Element = () => {
+  return (
+    <ExternalLink
+      url={"https://github.com/orgs/tenzir/projects/8/views/21"}
```

Comment:
```suggestion
      url={"https://github.com/orgs/tenzir/projects/8"}
```

This also works, and always points to the first view, which is a bit more robust.